oneview_redfish_toolkit/api/computer_system.py

```diff
@@ -16,8 +16,9 @@
 
 import collections
 from copy import deepcopy
+from flask_api import status
+from werkzeug.exceptions import abort
```

Comment:
You can use: from flask import abort
You could use this same library on others changes as well (Just to not repeat the same comment)